src/resources/template.ts.mustache

```diff
@@ -0,0 +1,47 @@
+import { QueryConfig as PgQuery } from 'pg';
+
+export const argumentPattern = {{{ argumentPattern }}};
+export const rawQuery = `{{{ rawQuery }}}`;
+
+export class MissingValueError extends Error {
```

Comment:
Recommend moving this error class out to a shared location, and importing/referencing it in the generated modules.

Reply:
I've thought about that, but didn't want to introduce a runtime dependency for downstream applications - I suppose, though, now with #4 introducing an extra file, writing a `common.ts` into the `generated/` folder would be pretty easy and alleviate this concern.

I'll go ahead and add that to #4!

Reply:
Implemented in #4 - https://github.com/logixboard/sureql/pull/4/commits/225396f26ef39ff198862dc6316d1fa478915f90